example/template.d/amanda-harddisk.conf.in

```diff
@@ -12,7 +12,7 @@ tapecycle 10 tapes             # the number of tapes in rotation
 runtapes 1                     # number of tapes to be used in a single run of amdump
 
 define changer my_vtapes {
-    tpchanger "chg-disk:/@prefix@/var/lib/amanda/vtapes/@DEFAULT_CONFIG@"
+    tpchanger "chg-disk:@localstatedir@/lib/amanda/vtapes/@DEFAULT_CONFIG@"
```

Comment:
isn't localstatedir typically /usr/lib/amanda?  So this would be /usr/lib/amanda/lib/amanda/vtapes?

Reply:
Everywhere I've seen ... local-state implies large files and variation over time.  That's always /var as I've seen it (e.g. /var/spool, /var/log, /var/cache etc...).  

There's still a long-leftover penchant of making /usr nearly unchanging except for resource files and the rest of the OS install in all the UNIXes I've remembered.   Some had it via NFS.

Reply:
Oh, good point.. OK!

Reply:
Yeah, I was totally off-base.  Per https://www.gnu.org/prep/standards/html_node/Directory-Variables.html localstatedir is typically $prefix/var, so this substitution is fine.